changelog/unreleased/enhancement-origin-event-check

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add origin check to Draw.io events
```

Comment:
Please change `Enhancement` to `Bugfix`. We handle security patches as bugfixes. :-) (filename respectively)